azext_edge/edge/params.py

```diff
@@ -858,6 +868,73 @@ def load_iotops_arguments(self, _):
                 ),
             )
 
+    with self.argument_context("iot ops registry") as context:
+        context.argument(
+            "instance_name",
+            options_list=["--instance", "-i"],
+            help="IoT Operations instance name.",
+        )
+        context.argument(
+            "registry_endpoint_name",
+            options_list=["--name", "-n"],
+            help="Registry endpoint name.",
+        )
+        context.argument(
+            "host",
+            options_list=["--host"],
+            help="The URL endpoint of the Azure Container Registry.",
+        )
+        context.argument(
+            "auth_type",
+            options_list=["--auth-type"],
+            arg_type=get_enum_type(RegistryEndpointAuthenticationType, default=None),
+            help="The authentication type for the registry endpoint. If not provided, "
+            "the authentication type will be determined based on the provided authentication parameters. "
+            "If no authentication parameters are provided, anonymous authentication will be used.",
+        )
+        context.argument(
+            "secret_ref",
+            options_list=["--secret-ref"],
+            help="Kubernetes secret reference for registry authentication.",
+            arg_group="Artifact Pull Secret",
+        )
+        context.argument(
+            "audience",
+            options_list=["--audience", "--aud"],
+            help="Audience for system-assigned managed identity registry authentication.",
+            arg_group="System-Assigned Identity",
+        )
+        context.argument(
+            "client_id",
+            options_list=["--client-id", "--cid"],
+            help="Client ID for user-assigned managed identity registry authentication.",
+            arg_group="User-Assigned Identity",
+        )
+        context.argument(
+            "tenant_id",
+            options_list=["--tenant-id", "--tid"],
+            help="Tenant ID for user-assigned managed identity registry authentication.",
+            arg_group="User-Assigned Identity",
+        )
+        context.argument(
+            "scope",
+            options_list=["--scope"],
+            help="Scope for user-assigned managed identity registry authentication.",
+            arg_group="User-Assigned Identity",
+        )
+        context.argument(
+            "trusted_signing_configmap_key",
+            options_list=["--trust-config-map", "--tcm"],
+            help="Trusted signing config map reference.",
+            arg_group="Trusted Signing",
+        )
+        context.argument(
+            "trusted_signing_secret_key",
+            options_list=["--trust-secret", "--ts"],
```

Comment:
Sometimes a ref ends with -ref other times not. Ideally its consistent, but there are inconsistencies across command spaces.